src/parser/jobs/rpr/changelog.tsx

```diff
@@ -3,6 +3,11 @@ import CONTRIBUTORS from 'data/CONTRIBUTORS'
 import React from 'react'
 
 export const changelog = [
+	{
+		date: new Date('2022-04-24'),
+		Changes: () => <>Reword Harvest Moon text to make Soulsow's use clearer.</>,
```

Comment:
Use clearer... what? :P

Reply:
Like, to make the usage of Soulsow clearer. Maybe an Australianism, it makes sense to me. Will update.